cmd/agent/app/integrations.go

```diff
@@ -523,7 +523,7 @@ func moveConfigurationFilesOf(integration string) error {
 		check = strings.Replace(check, "-", "_", -1)
 	}
 	confFileDest := filepath.Join(confFolder, fmt.Sprintf("%s.d", check))
-	if err := os.MkdirAll(confFileDest, os.ModeDir); err != nil {
+	if err := os.MkdirAll(confFileDest, os.ModeDir|0755); err != nil {
```

Comment:
Just checking: are these the intended permissions?

Reply:
Confirmed it works.
Also from go docs https://golang.org/pkg/os/#FileMode: 
> A FileMode represents a file's mode and permission bits. The bits have the same definition on all systems, so that information about files can be moved from one system to another portably. Not all bits apply to all systems. 
So no surprise there 🙂

Reply:
FWIW, the bitmask is still valid on Win but will have no effect on permissions (see how `mode` is ignored here https://github.com/golang/go/blob/50bd1c4d4eb4fac8ddeb5f063c099daccfb71b26/src/syscall/syscall_windows.go#L410). To enforce perms you need ACE/ACLs.